app/controllers/queued_videos_controller.rb

```diff
@@ -0,0 +1,36 @@
+class QueuedVideosController < ApplicationController
+  before_action :require_user
+
+  def index
+    @queued_videos = current_user.queued_videos
+  end
+
+  def create
+    queue_video(params[:video_id])
+    redirect_to queued_videos_path
+  end
+
+  def destroy
+    queued_vid = QueuedVideo.find(params[:id])
+    queued_vid.delete if queued_vid.user == current_user
+    redirect_to queued_videos_path
+  end
+
+  private
+  def queue_video(video_id)
+     if video_not_yet_queued?(video_id)
+      QueuedVideo.create(order: new_order,
```

Comment:
Now that I look closer I see it has a single space, you just missed the second space.